man/adoc/bpftrace.adoc

```diff
@@ -3427,6 +3427,22 @@ Default: 0
 Maximum number of levels of nested field accesses for tracepoint args.
 0 is unlimited.
 
+=== missing_probes
+
+Default: `warn`
+
+Controls handling of probes with multiple kprobe or uprobe attach points which
+cannot be attached to some functions because they do not exist in the kernel or
+in the traced binary.
+
+The possible options are:
+- `error` - always fail on missing probes,
```

Comment:
nit: don't need commas at the end of these I think or a period at the end of the last one

Reply:
Ah, that's my inner typography geek putting these in.